### DISCUSSION TITLE
How to specify scraping a single user (when subscribed to multiple)? Can't find that in ReadMe

### DISCUSSION THREAD

**Discussion by paulescobar** [UltimaHoarder/UltimaScraper#634] [Q&A] [answered]
I have multiple subscriptions. If I want to scrape just one OF profile, how can I specify that in the config file? I checked the ReadMe and it doesn't say.

**Comment by James-MF-Thomas** [comment, marked as answer]
turn "auto scrape names": to false

**Reply by paulescobar** [reply]
But after that, how would we specify a particular OF profile? I haven't executed the app yet. So I'm wondering what to expect.

**Reply by James-MF-Thomas** [reply]
you'll see all the models in paragraph form  with a number before each one. type the number of the corresponding model you want.

**Reply by James-MF-Thomas** [reply]
execute the script

**Reply by paulescobar** [reply]
Woops! I was confused because when I first ran the script and typed the OF profile name...it said "nothing to scrape". This time I entered "0". Then it provided a list of OF profiles to choose from. And now I can select the specific OF profile by entering the corresponding number associated with it.

**Reply by UltimaHoarder** [reply]
You can replace auto_scrape_name with a username instead

**Comment by clouds321** [comment]
I also have this issue. Using latest version, "auto scrape names": to false. I pick a number of a model, after finishing that model it will download others.

**Reply by ernesto-guevara-187** [reply]
^Same thing is happening to me, I chose the number corresponding to the model I want but it starts to download from all.

**Reply by UltimaHoarder** [reply]
https://github.com/DIGITALCRIMINAL/OnlyFans/issues/1106#issuecomment-830926185

Try

**Reply by ernesto-guevara-187** [reply]
Thank you! That seems to have done the trick!

And, just to be sure, what if anything am I/are we missing by setting that flag?

**Reply by UltimaHoarder** [reply]
> Thank you! That seems to have done the trick!
> 
> And, just to be sure, we're not missing any paid content by setting that to false?

If you're downloading a model normally (picking who you're downloading), you won't miss out on paid content.
